### ISSUE TITLE
Module Names with Underscores

### ISSUE THREAD

**Issue by jprince14** [oldani/HtmlTestRunner#29]
* HtmlTestRunner version: 1.1.2
* Python version: 3.6.4
* Operating System: Windows 10

When unittest module names contain underscores the name in the test report (within the html file) is only everything prior to the first underscore..

```
import HtmlTestRunner
import unittest

class example_name_with_underscore(unittest.TestCase):
    """the test report for this module will contain the name example within the html file"""

    def test_upper(self):
        self.assertEqual('foo'.upper(), 'FOO')

if __name__ == '__main__':
    suite = unittest.TestLoader().loadTestsFromTestCase(example_name_with_underscore)
    runner = HtmlTestRunner.HTMLTestRunner(output='example_dir')
    runner.run(suite)
```

**Comment by oldani** [comment]
Taking a look

**Comment by jprince14** [comment]
I think the cause is the line below in the function _report_tests

`        testcase_name = test_class_name.split("_")[1]
`

**Comment by oldani** [comment]
Fixed by @JamesMTSloan. Released on version `1.2`
